prov/gni/src/gnix_msg.c

```diff
@@ -208,6 +208,52 @@ static void __gnix_msg_pack_data_from_iov(uint64_t dest, size_t dest_len,
 	}
 }
 
+
+/*
+ * TODO: this can eventually be used in more places in this
+ * but the msg component of the fab request structure would
+ * need to be refactored.
+ */
+static inline int __gnix_msg_register_iov(struct gnix_fid_domain *dom,
+					  const struct iovec *iov,
+					  size_t count,
+					  struct gnix_fid_mem_desc **md_vec)
+{
+	int i, ret = FI_SUCCESS;
+	struct fid_mr *auto_mr;
+
+	for (i = 0; i < count; i++) {
+		auto_mr = NULL;
```

Comment:
Yeah this is probably not necessary within the loop.  Probably setting to NULL in the declaration may reduce likelihood of compiler warnings.  I'll fix this.